src/atoms/dialog/component.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { ISingleAnyChildProps, useEventListener } from '@kibalabs/core-react';
+import styled from 'styled-components';
+
+import { Box } from '../../particles/box';
+
+const StyledBackdrop = styled.div`
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.8);
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  left: 0;
+  top: 0;
+`;
+
+interface IDialogProps extends ISingleAnyChildProps {
+  isOpen: boolean;
+  maxHeight?: string;
+  maxWidth?: string;
+  isScrollableVertically?: boolean;
+  isScrollableHorizontally?: boolean;
+  onCloseClicked: () => void;
+}
+
+export const Dialog = (props: IDialogProps): React.ReactElement | null => {
+  const dialogRef = React.useRef();
```

Comment:
what's this for?

Reply:
this is for when we click the backdrop, the dialog should close